### PR TITLE
Changes in getting started

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -4,16 +4,20 @@
...
 The virtual machines are scheduled on the development machine.
 Foreman development environment is located on the development machine, created by following [Foreman's guide](https://theforeman.org/contribute.html) .
 
-Two libvirt networks are defined:
+Two libvirt networks should be defined:
 * default - provides external connectivity
 * foreman - inner network that connects the VMs (foreman <--> kubevirt <--> foreman-proxy). The network will serve for booting KubeVirt's VM from a PXE server.
   * Address on development machine: 192.168.111.1
-  * The domain *example.tst*
+  * The domain *kubevirt.com*
```
Comment (masayag) [inline docs/getting-started.md:15]: this domain actually exists and might lead to unexpected issues. perhaps kubevirt.tst ?

Comment (shiramax) [reply docs/getting-started.md:15]: it worked for me, but if you want I can change it.

Comment (masayag) [reply docs/getting-started.md:15]: let's change it through the guide

Comment (shiramax) [reply docs/getting-started.md]: done

File: docs/getting-started.md [hunk 2]
```diff
@@ -155,13 +270,13 @@ kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEV
 
 # before moving to next step, please make sure that kubevirt pods are running:
 kubectl get pods -n kubevirt -A
-kubectl apply -f https://raw.githubusercontent.com/kubevirt/ovs-cni/master/examples/ovs-cni.yml
```
Comment (masayag) [inline docs/getting-started.md:158]: pls remove the opposite (delete) command from the `Purge KubeVirt and Kubernetes` section.

File: docs/getting-started.md [hunk 3]
```diff
@@ -155,13 +270,13 @@ kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEV
...
 
 export NET_RELEASE=$(curl -s https://api.github.com/repos/kubevirt/cluster-network-addons-operator/releases | grep tag_name | grep -v -- - | sort -V  | tail -1 | awk -F':' '{print $2}' | sed 's/,//' | xargs)
 
 kubectl apply -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NET_RELEASE}/namespace.yaml
 kubectl apply -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NET_RELEASE}/network-addons-config.crd.yaml
 kubectl apply -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NET_RELEASE}/operator.yaml
+kubectl apply -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/${NET_RELEASE}/kubemacpool.yaml
```
Comment (masayag) [inline docs/getting-started.md:279]: pls add the opposite (delete) to the `Purge KubeVirt and Kubernetes`

**Review by masayag — CHANGES_REQUESTED** [review]
overall looks good to me, but have a small comment about purging the env.

**Comment by yifatmakias** [comment]
@masayag I added the requested changes but it opened me a new PR: https://github.com/theforeman/foreman_kubevirt/pull/142
so I am closing this one, take a look that nothing else is missing in the other PR.

**Pull request closed without merging**